python/GafferUI/ErrorDialogue.py

```diff
@@ -43,44 +43,102 @@
 
 class ErrorDialogue( GafferUI.Dialogue ) :
 
-	def __init__( self, title, message, details=None, **kw ) :
+	def __init__( self, title, message = None, details = None, messages = None, closeLabel = "Close", **kw ) :
```

Comment:
There are `message` and `messages` arguments now? Bit confusing... maybe a doc string if we can't think of more distinct naming?